Dockerfile

```diff
@@ -12,10 +12,32 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x | bash \
     && apt-get install --no-install-recommends -y nodejs \
     && rm -rf /var/lib/apt/lists/*
 
+# Kosmtik with plugins
 RUN npm install -g kosmtik
 
+WORKDIR /usr/lib/node_modules/kosmtik/
+RUN npm i kosmtik-overpass-layer
```

Comment:
Every RUN creates another layer (see https://stackoverflow.com/questions/31222377/what-are-docker-image-layers). Common practice is to group commands together. So why not writing `RUN npm i kosmtik-fetch-remote kosmtik-overlay ...`.